apply_extra.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/sh
```

Comment:
Any reason for not putting this in the manifest?

Reply:
Mostly just because there's a quite decent chance it'll grow over time, so it's easier to just pull it out now.